### ISSUE TITLE
[BUG] TypeError: '<' not supported between instances of 'datetime.datetime' and 'NoneType'

### ISSUE THREAD

**Issue by amrrs** [obsei/obsei#236]
**Describe the bug**

I'm not sure if this actually is a bug, but today I realized while using `lookup_period` I was getting this error 

```
TypeError: '<' not supported between instances of 'datetime.datetime' and 'NoneType'
```

**To Reproduce**
Steps to reproduce the behavior:https://github.com/amrrs/youtube-comment-sentiment-analysis/blob/main/Download_YouTube_Comments_NLP_Sentiment_Analysis.ipynb

**Expected behavior**
If it's a bug, suggest me how to fix it. 

Or if it's actually that the comments aren't there for those time period then we can show that kind of message. But in my case, there were comments during those period and yet I got that above error. 
 

**Please complete the following information:**
 - OS: Google Colab (Linux)
 - Version:

**Additional context**
Happy to give more details!

**Comment by amrrs** [comment]
I tried multiple videos here's one of them - https://www.youtube.com/watch?v=1Cw-vODp-8Y

**Comment by lalitpagaria** [comment]
Thank you @amrrs for reporting this. Let me have a look and get back to you.

**Comment by lalitpagaria** [comment]
@amrrs Yeah figured out this issue happened when Obsei fetch a comment which was edited. I added the fix in this commit https://github.com/obsei/obsei/commit/fffb7b15ea90f6794f1b6fcd9639b42ec80f7b9d

But it is currently not in the part of the released version. Soon we going to release a new version. Till then kindly install Osbei from the top of the branch.

**Comment by amrrs** [comment]
Thanks so much @lalitpagaria  for fixing it so soon!
